packages.yaml

```diff
@@ -172,6 +172,7 @@ packages:
       - tcl@8.6.6
       - tk@8.6.6 ^tcl@8.6.6
       - mercurial@4.1.2 ^python@2.7.13
+      - hisat2@2.1.0
```

Comment:
I see this in `hisat2` package:
```python
files = glob.iglob('*.py')
for file in files:
    if os.path.isfile(file):
        install(file, prefix.bin)
```
Did anybody check if the version of the python interpreter matters for these scripts, or if they are portable from 2.7.5 (system python) to 3.6.1 ?